package.json

```diff
@@ -167,6 +168,7 @@
 		"polyfill-library": "4.8.0",
 		"react": "18.2.0",
 		"react-dom": "18.2.0",
+		"react-is": "18.2.0",
```

Comment:
Do we miss `react-is` in the WordPress packages? What's the reason it had to be included here?

Reply:
Interesting, I didn't add it manually, so the scripts might have done it in some ways.

Reply:
That is very likely as the sync script tries to add missing WP dependencies, but maybe it goes beyond that.

Reply:
Maybe it's a peer dependency or something and something in the script or npm adds these automatically?

Reply:
Added in https://github.com/WordPress/gutenberg/pull/57193 by @Mamaduka. Everything looks good there.